lib/checks/aria/abstractrole-evaluate.js

```diff
@@ -1,6 +1,14 @@
 import { tokenList } from '../../core/utils';
```

Comment:
The `validate-scrollable-semantics` evaluate function simply checks, if a given node when scrollable, has the allowed html tag type and has a valid [landmark](https://www.w3.org/TR/wai-aria/#landmark_roles) role. The mapping of html tags vs allowed roles, I am unsure how this was arrived but can dig deeper to find out if necessary. I have seen this in WCAG docs and also in HTML Spec, when reading for ACT rules. This is a best practice rule.

In terms of documenting it, I would keep this simple:

```js
/**
 * Check if a given scrollable element has the valid semantics 
 * 
 * @param {HTMLElement} node
 * @return {Boolean}
 */
```